blog/releases/announcing-knative-v0-19-release.md

```diff
@@ -0,0 +1,283 @@
+---
+title: "Version v0.19 release"
+linkTitle: "Version v0.19 release"
+Author: "[Carlos Santana](https://twitter.com/csantanapr)"
+Author handle: https://github.com/csantanapr
+date: 2020-11-13
+description: "Knative v0.19 release announcement"
+type: "blog"
+image: knative-eventing.png
+---
+
+
+### Announcing Knative v0.19 Release
+
+A new version of Knative is now available across multiple components.
+Follow the instructions in the documentation [Installing Knative](https://knative.dev/docs/install/) for the respective component.
+
+#### Table of Contents
+- [Serving v0.19](#serving-v019)
+- [Eventing v0.19](#eventing-v019)
+- Eventing Extensions
+    - [Eventing Kafka Broker v0.19](#eventing-kafka-broker-v019)
+    - [Eventing Gitlab v0.19](#eventing-gitlab-v019)
```

Comment:
Did you also want the RabbitMQ bits in the TOC?

Reply:
Yes good catch, I missed rabbit mq I Will add

Reply:
@evankanderson thanks for the review I fix the TOC, but will keep it on hold until the docs are released for v0.19 are public